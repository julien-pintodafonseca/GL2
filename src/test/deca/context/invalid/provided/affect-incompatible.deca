// Description:
//      Affectation et typage
//
// Resultats:
//      Erreur contextuelle
//      Ligne 13 : Affectation de types incompatibles

{
   int a;
   boolean b;

   // Doit etre refuse.
   a = b;
}
