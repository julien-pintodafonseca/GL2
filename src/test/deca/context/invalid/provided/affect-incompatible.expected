affect-incompatible.deca:13:7: Le type de variable suivant est incompatible pour l'affectation : boolean ( type attendue : int )
